pkgs/servers/home-assistant/custom-components/easee/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+  pyeasee,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "nordicopen";
+  domain = "easee";
+  version = "0.9.67";
+
+  src = fetchFromGitHub {
+    owner = "nordicopen";
+    repo = "easee_hass";
+    tag = "v${version}";
+    hash = "sha256-psRr3cJ/sK/Z0dgB27GbW0qAHH2vJt+TdxqDB+Zhkc0=";
+  };
+
+  dependencies = [
+    pyeasee
+  ];
+
+  meta = {
+    description = "Custom component for Easee EV charger integration with Home Assistant";
+    homepage = "https://github.com/nordicopen/easee_hass";
+    changelog = "https://github.com/nordicopen/easee_hass/releases/tag/v${version}";
+    license = lib.licenses.unfree;
```

Comment:
Can you file an issue? Most often this is just an oversight in the home-assistant ecosystem. Because how can HACS distribute them otherwise?

Reply:
Done in https://github.com/nordicopen/easee_hass/issues/543 and they were quick to follow up with https://github.com/nordicopen/easee_hass/commit/2136ad8ec73a9cf87953b39470a6609f79ff770c

Of course the release packaged here hasn't been released under MIT yet, and the addition of the license was done without any public display of consent from the other authors, but that's pretty par for the course with these projects I guess.

Should I just mark it as MIT regardless?

Reply:
I'd suggest to wait for the first version that has the license to be released then bump this PR to package that.